core/Cadr.carp

```diff
@@ -0,0 +1,29 @@
+(defmodule Dynamic
```

Comment:
Should we just add these to `List.carp` as well?

Reply:
There was some dependency that made it hard I think, but I'll try again.